package.json

```diff
@@ -35,6 +37,7 @@
     "angular-animate": "^1.6.6",
     "angular-mocks": "^1.6.6",
     "angular-route": "^1.6.6",
-    "font-awesome": "^4.7.0"
+    "font-awesome": "^4.7.0",
+    "global": "^4.3.2"
```

Comment:
need to remove this global.  bad install when trying to install protractor globally with yarn. worked with npm though. need to check that